docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-controller
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a PKI, config, and database by installing interactively with prompts or by setting these values.
+
+1. In `/opt/openziti/etc/controller/service.env` (the answer file for the service)
+    1. Set `ZITI_BOOTSTRAP=true`
+1. In `/opt/openziti/etc/controller/bootstrap.env` (the answer file for bootstrapping the controller)
+    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller
+    1. Set `ZITI_PWD` to the desired management password for user `admin`. You may delete this after bootstrapping.
+
+### Generating a Public Key Infrastructure
+
+The controller service will generate a root CA and issue an intermediate CA cert during the first startup.
+
+The controller needs a CA to issue certificates for edge identities during enrollment. This CA is known as the edge signer. It's a good idea to secure the root in a different location and use an intermediate CA for this purpose. This allows you to recover in a scenario where the intermediate was lost without changing the root of trust and invalidating all edge enrollments.
+
+Disable bootstrapping a PKI by setting `ZITI_BOOTSTRAP_PKI=false` in `/opt/openziti/etc/controller/service.env`.
+
+Check out [the PKI page](/learn/core-concepts/pki.md) for an overview of these concepts.
+
+### Generating a Configuration
+
+The controller service will generate a configuration during the first startup.
+
+The controller's configuration is loaded from a YAML file ([reference](/reference/30-configuration/controller.md)). The Linux system service will generate a valid configuration during the first startup unless one already exists. 
+
+The filename of the configuration file, relative to the Linux service's working directory (`/var/lib/ziti-controller`), is specified in `/lib/systemd/system/ziti-controller.service` as an argument to the `ExecStart` command. You may override this in `/etc/systemd/system/ziti-controller.service.d/override.conf`.
+
+Disable bootstrapping a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in `/opt/openziti/etc/controller/service.env`.
+
+### Bootstrapping the Database
+
+The controller service will bootstrap a database during the first startup.
+
+The controller requires a BoltDB database to store its state. The Linux system service will initialize a database with a default admin password during the first startup unless one already exists.
+
+You must specify the management password for the default admin user before starting the service. This is done by setting `ZITI_PWD` in `/opt/openziti/etc/controller/bootstrap.env` or in the root-only file specified by `LoadCredential` in `/etc/systemd/system/ziti-controller.service.d/override.conf`. You may delete the password after bootstrapping for security.

```

Comment:
this fits in with the prior comment wrt implementation. the process should just remove the answer file for you imo and this section would just spell out that the file is removed after startup. Then "You may delete the password after bootstrapping for security." becomes "The answer file with the password is removed for you on successul configuration for security." (or something like that)

Reply:
raised an issue to redress how db bootstrapping works in https://github.com/openziti/ziti/issues/2000